src/drivers/uavcan/sensors/gnss.cpp

```diff
@@ -304,7 +319,7 @@ UavcanGnssBridge::gnss_fix2_sub_cb(const uavcan::ReceivedDataStructure<uavcan::e
 	uint8_t jamming_state = 0;
 	uint8_t spoofing_state = 0;
 
-	// Use ecef_position_velocity for now... There is no heading field
+	// Use ecef_position_velocity for CANNODE until FIXME above resolved
 	if (!msg.ecef_position_velocity.empty()) {
 		if (!std::isnan(msg.ecef_position_velocity[0].velocity_xyz[0])) {
 			heading = msg.ecef_position_velocity[0].velocity_xyz[0];
```

Comment:
I think if `_rel_heading_valid` we should not use this field for heading

Reply:
fixed